Cargo.lock

```diff
@@ -3139,6 +3139,7 @@ dependencies = [
  "open",
  "public-ip",
  "rand 0.8.5",
+ "regex",
```

Comment:
is this crate still being used? i don't see it in the diff -- if not, please remove from Cargo.lock